src/MizarWidgets/SamplingWithFrameTrackWidget.cpp

```diff
@@ -4,7 +4,9 @@
 
 #include "MizarWidgets/SamplingWithFrameTrackWidget.h"
 
+#include <QObject>
 #include <QWidget>
+#include <Qt>
```

Comment:
I think this include is probably not needed? Maybe a more specific one

Reply:
Thanks. I thought it's needed for `Qt::CheckState`

Reply:
Yeah, I did not look this up. You are right, Qt::CheckState is part of the namespace and the correct include for this is <Qt>. I learned something :)

Reply:
Anyway, thanks to you I don't need Qt::CheckState anymore